godot-bindings/src/lib.rs

```diff
@@ -189,6 +219,28 @@ pub fn remove_dir_all_reliable(path: &Path) {
         }
     }
 }
+
+pub fn before_api(major_minor: &str) -> bool {
+    let mut parts = major_minor.split('.');
+    let queried_major = parts
+        .next()
+        .unwrap()
+        .parse::<u8>()
+        .expect("invalid major version");
+    let queried_minor = parts
+        .next()
+        .unwrap()
+        .parse::<u8>()
+        .expect("invalid minor version");
+    assert_eq!(queried_major, 4, "major version must be 4");
+    let godot_version = get_godot_version();
+    godot_version.minor < queried_minor
+}
+
+pub fn since_api(major_minor: &str) -> bool {
+    !before_api(major_minor)
+}
```

Comment:
This duplicates the code from `GdextBuild`, which is fine -- but maybe write a comment in both places, so that changes to one are synchronized with the other.

[Here](https://github.com/godot-rust/gdext/blob/master/godot-codegen/src/generator/gdext_build_struct.rs#L12-L12)'s where `GdextBuild` is generated.